unit_testing/readme.md

```diff
@@ -0,0 +1,10 @@
+### README
+
+This directory is for unit testing, which should insure that all pull requests have valid files.
+Each file in the form scriptname.test.js will be run by [jest](https://facebook.github.io/jest/docs/en/getting-started.html), a node based
+unit testing library created by facebook. To understand what unit testing does check out 
+[the coding trains video series](https://www.youtube.com/watch?v=CB7vnoXI0pE&list=PLRqwX-V7Uu6bLqwFa52YGEHy-L1-D_Ve-).
```

Comment:
Should we link to the site here instead of YouTube?
Also I think I'd prefer `The Coding Train's video series` but that's not terribly important

Reply:
I capitalized "The Coding Train's video series" as suggested.

Reply:
Not sure where the unit testing video series is on the site however, as I tend to watch the coding train on youtube (I did have a look but couldn't find it).